### PR TITLE
chore(install): Remove dependency check

### PR DESCRIPTION
**Pull request by lgeiger**
Follow up to #1268

### CODE REVIEW

File: scripts/check-deps.js [hunk 1]
```diff
@@ -1,43 +0,0 @@
...
-  depedencies.forEach(function(dep) {
-    commandExists(dep, function(err, commandExists) {
-      if(!commandExists) {
-        console.error(`Missing dependency: ${dep.bold} Could not be found on your system.`.red);
-      }
-    });
-  });
-
-  if (process.platform === 'linux') {
-    var publishDeps = ['icnsutils', 'graphicsmagick', 'xz-utils'];
```
Comment (rgbkrk) [inline scripts/check-deps.js:34]: We still need these though right?

Comment (lgeiger) [reply scripts/check-deps.js:34]: Yes. But I noticed that `icnsutils`, `graphicsmagick` and `xz-utils` aren't even detected by `commandExists` hence this check always failed.

Comment (lgeiger) [reply scripts/check-deps.js:34]: And they are only necessary for packaging a release.

Comment (rgbkrk) [reply scripts/check-deps.js:34]: Sounds good, let's go with the trim down.

**Pull request merged**